conda/run_travis_locally/build_mpich_libE.sh

```diff
@@ -7,13 +7,14 @@ set -x # problem with this - loads of conda internal commands shown - overwhelmi
 
 export PYTHON_VERSION=3.7       # default - override with -p <version>
 export LIBE_BRANCH="develop"    # default - override with -b <branchname>
-
+export SYSTEM="Linux"         # default - override with -s <OS>
```

Comment:
Which variables are options for Minconda?

Reply:
Line now merged